src/operations/set_trustline_flags.js

```diff
@@ -0,0 +1,59 @@
+import xdr from '../generated/stellar-xdr_generated';
+import { Keypair } from '../keypair';
+
+/**
+ * Creates a trustline flag configuring operation.
+ *
+ * @function
+ * @alias Operation.setTrustLineFlags
+ *
+ * @param {object} opts - Options object
+ * @param {string} opts.trustor     - the account whose trustline this is
+ * @param {Asset}  opts.asset       - the asset on the trustline
+ * @param {[xdr.TrustLineFlags]} opts.clearFlags - the flags to clear
+ * @param {[xdr.TrustLineFlags]} opts.setFlags   - the flags to set
```

Comment:
I not sure I understand exactly the approach you're proposing, please let me know if I misunderstand.

If you're suggesting that `authorized: true` set the flag, `authorized: false` clear the flag, and `authorized: null` or `authorized: undefined` does nothing, then I think that works, but it is less clear.

Personally I'd go the clear approach of setFlags and clearFlags with enums, but I might be a bit too close to the implementation, and I'm not a JS coder by day, so I defer to @stellar/horizon-committers @quietbits @piyalbasu @stellar/vibrant-mobile and others.

Reply:
Yeah, that's exactly my proposal; I thought it was a more intuitive way to think about flags (true -> on, false -> off). I can keep the design as-is, of course, where users just list what they want to set or clear with the fully-qualified XDR enum, but unfortunately that's [so verbose](https://github.com/stellar/js-stellar-base/pull/397/files#diff-583d150626144f6cb47c5138df52906bfd994df780d6c36348bb5b9c94606dfaR2219-R2227), and replicating the flags in a shorter form begs for bugs down the line if they diverge.

Reply:
The `true`, `false`, `null`|`undefined`, approach does seem much more succinct. I think that is 👍🏻 .

Reply:
Done in e2df0e836f40e0861b018719db02fe8c440db6b3; thoughts?

Reply:
I would name `maintainLiabilities` more completely as `authorizedToMaintainLiabilities`, because that is the trustline flag name, but otherwise I think this looks good. Good call.

Reply:
Sure, done!